src/checkers/inference/model/serialization/ToStringSerializer.java

```diff
@@ -24,49 +28,65 @@
 import checkers.inference.model.VariableSlot;
 
 /**
- * This Serializer is meant only to convert constraints and variables to
- * human readable strings.  It is used currently by the DebugSolver.
+ * This Serializer is meant only to convert constraints and variables to human readable strings. It
+ * is used currently by the DebugSolver.
```

Comment:
Can we remove the "currently by the DebugSolver"? I remember prof said Javadoc shouldn't mention its clients.